openapi.yml

```diff
@@ -281,6 +281,51 @@ paths:
           description:  Can´t change data. Circuit is deleted and archived.
         '404':
           description:  Schedule id not found.
+
+  /v2/evc/metadata:
+    post:
+      summary: Add metadata to a bulk of EVCs
+      description: Add metadata to multiple EVCs based on the body content
+      operationId: bulk_add_metadata
+      responses:
+        '200':
+          description: Operation successful
+        '400':
+          description: The request body is not a well-formed JSON or circuit_ids is missing
+        '404':
+          description: There were EVCs not found.
+          content:
+            aplication/json:
+              schema:
+                type: array
```

Comment:
Can you fix the spec err? To double check if it's fixed you can also use [their online editor](https://editor.swagger.io/) or use local swagger tools

```
Semantic error at paths./v2/evc/metadata.post.responses.404.content.aplication/json.schema
Schemas with 'type: array', require a sibling 'items: ' field
Jump to line 299
```

Reply:
Fixed